src/library/scala/collection/mutable/WrappedArray.scala

```diff
@@ -114,13 +114,44 @@ object WrappedArray {
     case x: Array[Unit]    => new ofUnit(x)
   }).asInstanceOf[WrappedArray[T]]
 
-  implicit def canBuildFrom[T](implicit m: ClassTag[T]): CanBuildFrom[WrappedArray[_], T, WrappedArray[T]] =
+  implicit def canBuildFrom[T](implicit m: ClassTag[T]): CanBuildFrom[WrappedArray[_], T, WrappedArray[T]] = {
+    val tag = implicitly[ClassTag[T]]
+    (tag.runtimeClass match {
+      case java.lang.Byte.TYPE      => cbfByteArray
+      case java.lang.Short.TYPE     => cbfShortArray
+      case java.lang.Character.TYPE => cbfCharArray
+      case java.lang.Integer.TYPE   => cbfIntArray
+      case java.lang.Long.TYPE      => cbfLongArray
+      case java.lang.Float.TYPE     => cbfFloatArray
+      case java.lang.Double.TYPE    => cbfDoubleArray
+      case java.lang.Boolean.TYPE   => cbfBooleanArray
+      case java.lang.Void.TYPE      => cbfUnitArray
+      case objectClass              => cbfObjectArray
+      case _ => makeCBF[T with AnyRef]
```

Comment:
Why `with AnyRef`? I think that's why the bootstrapped compilation is failing.

Reply:
seemed implied in IDE. Removed and updated objectClass to ObjectClass